airflow/models/log.py

```diff
@@ -40,11 +40,13 @@ class Log(Base):
     owner = Column(String(500))
     owner_display_name = Column(String(500))
     extra = Column(Text)
+    try_number = Column(Integer)
```

Comment:
This should probably have nullable=False?

Reply:
all the TI attrs must be nullable because not every log record in this table has something to do with a TI